### PR TITLE
Add CocoaPods link in README

### PR DESCRIPTION
**Pull request by ReadmeCritic**
For those who are not so clear what this `pod` business is 😄

### CODE REVIEW

**Comment by roundhill** [comment]
Maybe we should add Xcode and pods to the `Development Requirements` section?

**Comment by ReadmeCritic** [comment]
Good idea, will revise

**Comment by ReadmeCritic** [comment]
@roundhill take a look 👀

**Comment by roundhill** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 A Simplenote client for iOS. Learn more about Simplenote at [Simplenote.com](https://simplenote.com).
 
 ## Development Requirements
-* A Simperium account. [Sign up here](https://simperium.com/signup/)
-* A Simperium Application ID and key. [Create a new app here](https://simperium.com/app/new/)
+* A Simperium account ([sign up here](https://simperium.com/signup/)).
+* A Simperium Application ID and key ([create a new app here](https://simperium.com/app/new/)).
```
Comment (foadhassanzadeh) [inline README.md]: @@ -2,8 +2,10 @@
 A Simplenote client for iOS. Learn more about Simplenote at [Simplenote.com](https://simplenote.com).
 
+* A Simperium account ([sign up here](https://simperium.com/signup/)).
+* A Simperium Application ID and key ([create a new app here](https://simperium.com/app/new/)).
+* [CocoaPods](https://cocoapods.org/).
+* [Xcode](https://developer.apple.com/xcode/).
 
 ## Running

**Pull request merged**